dataverse-webapp/src/main/java/edu/harvard/iq/dataverse/api/Access.java

```diff
@@ -601,6 +607,14 @@ public void write(OutputStream os) throws IOException,
 
                                         zipper.addToManifest(fileName + " (" + mimeType + ") " + " skipped because the total size of the download bundle exceeded the limit of " + zipDownloadSizeLimit + " bytes.\r\n");
                                     }
+                                } else if (isRestrictedByEmbargo(file.getOwner())) {
+                                    if (zipper == null) {
+                                        fileManifest = fileManifest + "File with id=" + file.getId() +
+                                                " IS EMBARGOED UNTILL "+ file.getOwner().getEmbargoDate().get().toInstant().atOffset(ZoneOffset.UTC) + "\r\n";
```

Comment:
If we decide that acces is restricted by embargo and embargo is not present I think we can safely assume that something really wrong happened in the app and a `NullPointerException` - unlikely as it is - would be a decent indicator that something bad happened.
I felt like giving user only something like `Response.Status.FORBIDDEN` would be not informative enough but still I can settle on that.

Reply:
null pointer could be sufficient enough but not for the api user. The user would receive: 

> {"status":"ERROR","code":500,"message":"Internal server error. More details available at the server logs.
The user will not know what he did wrong. Imo there should be some response that says that embargo date is not present or something.

Reply:
That wouldn't be helpful for user neither. Something like 

> File is restricted by embargo but embargo is not present <

Is no more informative for user than knowledge that app broke. 
IMO we should either don't give user info about embargo date or don't worry about not-really-possible internal server errors and give him the date. This is such a narrow corner case that breaks literally nothing that I'm not sure if the effort is worth it.

Reply:
Alright that's true, but now that i think about null pointer could not be enough info, since we wouldn't know for which file it was. Fool proof version would be getOrElseThrow with exception that has file id attached to message.